src/components/matrix/Matrix.java

```diff
@@ -0,0 +1,117 @@
+package src.components.matrix;
+
+//Hello Jeremy
+
+public class Matrix {// extends components.standard<Matrix> {
+    private int[][] rep;
+
+    private void createNewRep(int i, int j) {
+        this.rep = new int[i][j];
+
+        for (int x = 0; x < i; x++) {
+            for (int y = 0; y < j; y++) {
+                this.rep[x][y] = 0;
+            }
+        }
+    }
+
+    public Matrix(int[][] m) {
+        this.rep = m.clone();
+    }
+
+    public Matrix(int i, int j) {
+        this.createNewRep(i, j);
+    }
+
+    public int get(int i, int j) {
```

Comment:
`get` is a fine name, but I might be slightly more descriptive with something like `getElement` or `getCell`.